main.nf

```diff
@@ -563,6 +563,7 @@ process search_engine_msgf {
                      -max_precursor_charge ${params.max_precursor_charge} \\
                      -min_peptide_length ${params.min_peptide_length} \\
                      -max_peptide_length ${params.max_peptide_length} \\
+		     -max_missed_cleavages ${params.allowed_missed_cleavages} \\
```

Comment:
indentation seems off? + think this requires an addition to the schema?

Reply:
Indent yes, schema no. It is already there. It was just not usable for this specific search engine until recently. It is used and described for comet already.